packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -453,10 +453,24 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
       await this.stopPromise
     }
 
+    const retries = { times: 10 }
+    const retryInternalStart = async () => {
+      try {
+        await this.internalStart()
+      } catch (e) {
+        if (e.retryable === true && retries.times > 0) {
+          retries.times--
+          await retryInternalStart()
+        }
+
+        throw e
+      }
+    }
```

Comment:
Is it missing a `retryable = false` somewhere here?

Reply:
Is that what you meant to ask?

Reply:
Does that answer your question?

Reply:
Looks good!